test/simple/flake.nix

```diff
@@ -51,8 +51,12 @@
               jailbreak = true;
               cabalFlags.blah = true;
             };
-            # Test STatic ANalysis report generation
-            haskell-flake-test.stan = true;
+            haskell-flake-test = {
+              # Test STatic ANalysis report generation
+              stan = true;
+              # Test if user's setting overrides the `jailbreak = false;` override by `buildFromSdist`
+              jailbreak = true;
```

Comment:
Add a comment here saying what exactly is being jailbroken.

Reply:
https://github.com/srid/haskell-flake/pull/420/commits/1a4f4f6ea1ac6c87c238cbfd72cf8d2358b5a3ec